crates/astria-conductor/src/executor.rs

```diff
@@ -68,15 +64,13 @@ pub(crate) async fn spawn(conf: &Config, alert_tx: AlertSender) -> Result<(JoinH
     Ok((join_handle, executor_tx))
 }
 
-// Given a string, convert to protobuf timestamp
-fn convert_str_to_prost_timestamp(value: &str) -> Result<ProstTimestamp> {
-    let time =
-        Time::parse_from_rfc3339(value).wrap_err("failed parsing string as rfc3339 datetime")?;
+// Given `Time`, convert to protobuf timestamp
+fn convert_str_to_prost_timestamp(value: Time) -> Result<ProstTimestamp> {
```

Comment:
Incorrect function name: not converting strings anymore and this can't fail.

```suggestion
fn convert_tendermint_to_prost_timestamp(value: Time) {
```